src/pages/ArticleDetailPage.tsx

```diff
@@ -122,13 +124,15 @@ const ArticleDetailPage = () => {
             {image && (
               <img
                 alt={articleTitle}
-                src={image}
+                src={getOptimizedImageURL({ url: image, width: 350, height: 320 })}
                 className={
                   !isImageLoaded
                     ? 'h-full m-5 rounded-lg bg-gray-300 dark:bg-gray-800 animate-pulse'
                     : 'h-full object-scale-down m-5 rounded-lg'
                 }
                 onLoad={() => setIsImageLoaded(true)}
+                width="352px"
+                height="320px"
```

Comment:
p5: 이 부분은 오타인가요? (350, 320)
```suggestion
                width="350px"
                height="320px"
```

Reply:
width를 재보니까 351.xxx여서 고민하다가 둘을 다르게 쓴 것 같네요ㅎㅎ 수정햐겠습니다